docs/guides/graphql.md

```diff
@@ -0,0 +1,397 @@
+---
+title: Building a GraphQL API with Nitric
+description: Use the Nitric framework to easily build and deploy a serverless GraphQL API for AWS, Google Cloud, or Azure
+---
+
+## What we'll be doing
+
+[GraphQL](https://graphql.org) APIs rely on only one HTTP endpoint, which means that you want it to be reliable, scalable, and performant. By using serverless compute such as Lambda, the GraphQL endpoint can be auto-scaling, whilst maintaining performance and reliability.
+
+We'll be using Nitric to create a GraphQL API, that can be deployed to a cloud of your choice, gaining the benefits of serverless compute.
+
+1. Create the GraphQL Schema
+2. Write Resolvers
+3. Create handler for GraphQL requests
+4. Run locally for testing
+5. Deploy to a cloud of your choice
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- The [Nitric CLI](https://nitric.io/docs/installation)
+- An [AWS](https://aws.amazon.com), [GCP](https://cloud.google.com) or [Azure](https://azure.microsoft.com) account (_your choice_)
+
+## Getting Started
+
+## Build the GraphQL Schema
```

Comment:
These two headings are at the same depth (H2) with no content in between. Should "Build the GraphQL Schema" be H3 instead?